src/components/Client.ts

```diff
@@ -88,7 +88,7 @@ const createStream = () => {
 
 const createStreamInfo: (
   context: ExtensionContext
-) => () => Thenable<StreamInfo> = (context) => {
+) => () => Promise<StreamInfo> = (context) => {
```

Comment:
I'm curious why this changed?

Reply:
New version of the LanguageClient required a different (although really the same) type

Reply:
basically a promise satisifies Thenable but the client now requires a Promise explicitly